handbook/engineering/README.md

```diff
@@ -36,6 +36,20 @@ To [create an engineering-initiated user story](https://fleetdm.com/handbook/eng
 
 > We prefer the term engineering-initiated stories over technical debt because the user story format helps keep us focused on our users and contributors.
 
+### Manage release branches
+Every three weeks, release a minor version of Fleet from the `main` branch. 
+
+Every week between minor releases, release a patch release with fixes for released bugs. 
+
+The DRI for publishing a release is responsible for creating the next patch branch. Create a patch branch off the latest tagged release of Fleet. Submit PRs for released bug fixes directly to the target patch branch to avoid merge conflicts later in the release cycle. After merging into the patch branch, submit another PR to `main` containing the same fix and resolve any merge conflicts.
+
+> It is the responsibility of the person merging the fix into the patch branch to make sure the fix is also merged into `main`.
+
+### Fix a bug
+If the bug is labeled `~unreleased bug`, branch off and put your PR into `main`.
+
+If the bug is labeled `~released bug`, branch off and put your PR into the upcoming patch branch `patch-fleet-v4.x.x`. If you are unsure which branch to use, confirm with your manager. Don't forget to also submit a second PR to `main` after the bug is confirmed fixed.
```

Comment:
For the first few sprints, it might be worth calling out in the issue's description (in addition to the `released bug` label) that the fix needs to go to the patch branch first. It would be easy to miss (even for reviewers) and we could end up having to do the cherry-pick/fix conflicts thing again. I assume after a couple of sprints it will become more natural.

Reply:
Sounds good. 👍 
Need to craete 4.47.1

Reply:
@lukeheath We'll need to address also bug PR's that were already merged to main.

Reply:
@sharon-fdm I've created the milestone. Please work with your team to make sure PRs are put into the patch branch for any released bug fixes they've already merged since the freeze lifted.